Update hello-world-program.md (#425)
solana-foundation/developer-content: content/courses/native-onchain-development/hello-world-program.md

## Summary

- Native Solana programs have a single **entry point** to process instructions

> - A program processes an instruction using the **program_id**, a list of
>   **accounts**, and **instruction_data** included with the instruction

Reviewer: ```suggestion
- A program processes an instruction using the **program_id**, a set of named
  **accounts**, and **instruction_data** included with the instruction
```

The path to `AccountInfo` would be `solana_program::account_info::AccountInfo`.

> Absent any other keywords, we would need to reference this entire path to use

Reviewer: ```suggestion
Without any `use` keywords, we would need to reference this entire path to use
```

#### 5. Invoke Program

> In this final step, we'll invoke the "Hello, world!" Solana program from the client side. While the main focus of this lesson is on building your Solana program, we've provided [the client code](https://github.com/Unboxed-Software/solana-hello-world-client) to help you easily interact with your deployed program.

Reviewer: Moved to https://github.com/solana-developers/hello-world-client - can you please make your updates there instead and fix these links?

Before we dive into the steps, watch the tutorial video below for a visual walkthrough:

![hello-world-solana](https://github.com/user-attachments/assets/6865e275-bbb9-43a1-acf7-8a97438c8ed5)

> 

Reviewer: Video is a good idea. The quality of the video is bad and there seems to be no sound - want to re-record and add a voiceover? 

Then contact me via telegram or twitter and send me the raw video for upload to our YouTube channel.
